release-team/hack/sync-bug-triage-github-project-beta.sh

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: `sync-bug-triage-github-project-beta.sh`
+# Pre-requisite: add a GITHUB_TOKEN as the environment variable (Ref: https://github.com/kubernetes/org/issues/3558)
+
+set -eu -o pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+ORGANIZATION="kubernetes"
+REPOSITORY="kubernetes"
+
+PROJECT_NUMBER=${GITHUB_PROJECT_BETA_NUMBER:-}
+MILESTONE=${RELEASE_MILESTONE:-}
+
+if [ "$PROJECT_NUMBER" == "" ]; then
+      echo "[Error] Required environment variable \"GITHUB_PROJECT_BETA_NUMBER\" is not set."
+      exit 1
+fi
+
+if [ "$MILESTONE" == "" ]; then
+      echo "[Error] Required environment variable \"RELEASE_MILESTONE\" is not set."
+      exit 1
+fi
+
+
+milestone_issue_ids=()
+milestone_pr_ids=()
+
+# function definitions
+function get_field_ids_from_github_beta_project() {
+    if [ $1 == "project-id" ]
+    then
+        query='.data.organization.projectV2.id'
+    elif [ $1 == "type-id" ]
+    then
+        query='.data.organization.projectV2.fields.nodes[] | select(.name== "Type") | .id'
+    elif [ $1 == "issue-id" ]
+    then
+        query='.data.organization.projectV2.fields.nodes[] | select(.name== "Type") | .options[] | select(.name== "Issue") | .id'
+    # for "pr-id"
+    else
+        query='.data.organization.projectV2.fields.nodes[] | select(.name== "Type") | .options[] | select(.name== "PR") | .id'
+    fi
+
+    ID="$( gh api graphql -f query='
+                    query($org: String!, $number: Int!) {
+                    organization(login: $org){
+                        projectV2(number: $number) {
+                        id
+                        fields(first:100) {
+                            nodes {
+                            ... on ProjectV2Field {
+                                id
+                                name
+                            }
+                            ... on ProjectV2SingleSelectField {
+                                id
+                                name
+                                options {
+                                    id
+                                    name
+                                }
+                            }
+                            }
+                        }
+                        }
+                    }
+                    }' -f org=${ORGANIZATION} -F number=${PROJECT_NUMBER} --jq "$query")"
```

Comment:
```suggestion
                    }' -f org="${ORGANIZATION}" -F number="${PROJECT_NUMBER}" --jq "$query")"
```

Adding double quotes to the other variables 👍

Reply:
addressed. Thank you!